### ISSUE TITLE
Hangs with Livewire and L9?

### ISSUE THREAD

**Issue by brandon-9** [tonysm/rich-text-laravel#21]
Trying to get this working on a Livewire component. But as soon as we submit the form, the whole page times out. 
It only times out when we try saving the related model. Remove that, and the form submits fine (but obviously no content saves).

We have `HasRichText` on the Account model. `about` is the field content saving.

```public function submit()
    {
        $this->validate();
        $this->account->about = 'test';
        $this->account->save();                  // remove this and everything processes fine.

        return redirect(route('settings.preferences.listing'));
    }

**Comment by tonysm** [comment]
Testing right now.

**Comment by tonysm** [comment]
Interesting. Looks like it's not playing really well with Livewire indeed. I'll need to spend more time with it, but from my tests I needed to refresh the instance bound to Livewire, something like this:

```php
$this->validate();

$this->user->update([
    'bio' => $this->bio,
]);

$this->user->refresh();
```

---

I'll test it more soon.

**Comment by tonysm** [comment]
Hey @brandon-9

I've just released version [1.4.1](https://github.com/tonysm/rich-text-laravel/releases/tag/1.4.1) with a fix for this problem. There's also a new section in the [readme related to Livewire](https://github.com/tonysm/rich-text-laravel#livewire). Please, take a look at that and let me know if it works for you. I've tested this fix as regular HTML forms and in a Livewire component.

I'll close the issue for now, but feel free to open it if you're still having problems with it.
